site/src/sphinx/server-sse.rst

```diff
@@ -49,3 +53,37 @@ use the annotation.
     public Publisher<ServerSentEvent> sse() {
         return Flux.just(ServerSentEvent.ofData("foo"), ServerSentEvent.ofData("bar"));
     }
+
+Adjusting the request timeout
+-----------------------------
+
+The events may being sent for a longer time depending on the application. Even it can continue infinitely,
```

Comment:
- The events -> An event stream
- being -> be
- a longer time -> a longer period than the configured timeout
- Even it -> It even